superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -264,6 +272,35 @@ const ResultSet = ({
         schema: query?.schema,
       };
 
+      // Antd >= 4.24.0 format:
+      const exportMenuItems = [];
+      exportMenuItems.push({
+        label: t('CSV'),
+        key: 'csv',
+        icon: <Icons.FileOutlined />,
+        onClick: () =>
+          window.open(getExportCsvUrl(query.id), '_blank')?.focus(),
+      });
+      if (isFeatureEnabled(FeatureFlag.GOOGLE_SHEETS_EXPORT)) {
+        exportMenuItems.push({
+          label: t('Google Sheets'),
+          key: 'google-sheets',
+          icon: <Icons.GoogleOutlined />,
+          onClick: () =>
+            window.open(getExportGoogleSheetsUrl(query.id), '_blank')?.focus(),
```

Comment:
This onClick handler should have the logic for gSheetApi request.
```suggestion
     const [trigger, { data, isLoading, isError }] = useLazyExportQuery();
     ---
     onClick: () => trigger({ clientId: getExportGoogleSheetsUrl(query.id) }),
```